core/core/ledger/pb/ledger.pb.go

```diff
@@ -0,0 +1,1101 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
This file is extra?

Reply:
Since they make changes on proto file, I have to generate new one too